src/essential_validator.py

```diff
@@ -392,38 +392,42 @@ def check_relationship(self, file_info, type, columns):
                 return True, None
             else: # check if invalid relationships
                 msg = f'Relationships {json.dumps(rel_props)} are not defined.' if len(rel_props) > 1 else f'Relationship "{rel_props[0]}" is not defined.'
-                return False, f'“{file_info[FILE_NAME]}”: {msg}'
+                return False, [f'“{file_info[FILE_NAME]}”: {msg}']
         
         # check if has relationship
         if len(rel_props) == 0:
-            return False, f'“{file_info[FILE_NAME]}”: No relationships specified.'
+            return False, [f'“{file_info[FILE_NAME]}”: No relationships specified.']
         def_rel_nodes = [ key for key in def_rel.keys()]
         rel_props_dic = {rel.split(".")[0]: rel.split(".")[1] for rel in columns if "." in rel}
         rel_props_dic_types = rel_props_dic.keys()
        
         #  check if missing relationship
         rel_missed = [node for node in def_rel_nodes if node not in rel_props_dic_types]
         if len(rel_props_dic_types) == 0:
-            msg = f'Relationships to parents, {json.dumps(rel_missed)}, are not specified.' if len(rel_missed) > 1 else f'Relationship to parent, "{rel_missed[0]}", is not specified.'
-            return False, f'“{file_info[FILE_NAME]}”: {msg}'
+            msg = f'Relationships to parents {json.dumps(rel_missed)} are not specified.' if len(rel_missed) > 1 else f'Relationship to parent "{rel_missed[0]}" is not specified.'
```

Comment:
I've never seen this message, will it still happen? What's the situation cause this error?

Reply:
updated